### PR TITLE
add: calculator play

### LINKED ISSUE #727: [Add a Play]: Calculator app
### What's the Play about?

- This is a calculator app play which would be able to perform basic arithmetic operations

### What ReactJS concept will be used to create this play?

- State mangement
- Event handling

### Are you willing to take it up for implementation?

Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by yashpaddalwar1234**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

This is a calculator app play which would be able to perform basic arithmetic operations

Fixes #727 

## Type of change

Please delete options that are not relevant.

- [ ] New Play (non-breaking change which adds functionality)

# How Has This Been Tested?
e2e and local testing
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### CODE REVIEW

**Comment by Angryman18** [comment]
this is most likely a javascript play not react play.

**Comment by harshsinghatz** [comment]
> this is most likely a javascript play not react play.

Can you explain a bit more?. Because this play uses JSX and hooks.

**Comment by Angryman18** [comment]
> > this is most likely a javascript play not react play.
> 
> Can you explain a bit more?. Because this play uses JSX and hooks.

no its fine. its good. but i see calling document object and modifying realdom at several places.

**Comment by harshsinghatz** [comment]
> > > this is most likely a javascript play not react play.
> > 
> > 
> > Can you explain a bit more?. Because this play uses JSX and hooks.
> 
> no its fine. its good. but i see calling document object and modifying realdom at several places.

Got it @Angryman18 !, but I guess this should work as well

**Review by koustov — APPROVED** [review]
lgtm

**Comment by koustov** [comment]
@Angryman18 are you good with this PR?

File: src/plays/calculator-app/CalculatorApp.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            handleAction(val);
+        }
+
+        if (type === "operation") {
+            handleNumberAndOperation(val);
+        }
+    }
+
+    return <div className='calc-app-container'>
+        <div className='calc-container'>
```
Comment (Angryman18) [inline src/plays/calculator-app/CalculatorApp.tsx:62]: where is PlayHeader component?

**Review by Angryman18 — CHANGES_REQUESTED** [review]

**Comment by Angryman18** [comment]
any update @harshsinghatz

**Pull request closed without merging**